### PR TITLE
Updating bootcamp information

### PR DESCRIPTION
**Pull request by janine9vn**
Adds time and more information about the GitHub bootcamp.

### CODE REVIEW

**Review by jb3 — APPROVED** [review]
I was told to approve this, I'm slightly intoxicated but it looks good. :trollface:

File: pydis_site/templates/events/pages/code-jams/8/github-bootcamp.html [hunk 1]
```diff
@@ -13,24 +13,29 @@
...
     <p>
-        This year we'll be running a GitHub Bootcamp on Saturday, June 26th.
+        <strong>This year we'll be running a GitHub Bootcamp on Saturday, June 26th at 4PM UTC.</strong>
         This bootcamp is intended to be an interactive workshop session where we help participants
-        get setup with git and using it in a team environment.
+        get setup with git and using it in a team environment. If you are new to git or would like to brush up on it,
+        then this is the perfect event for you.
+    </p>
+    <p>
+        The instructional parts of this bootcamp will be recorded. We'll also be providing a cheatsheet / reference guide to all attendees that will
```
Comment (Shivansh-007) [inline pydis_site/templates/events/pages/code-jams/8/github-bootcamp.html:22]:
```suggestion
        The instructional parts of this Bootcamp will be recorded. We'll also be providing a cheatsheet/reference guide to all attendees that will
```
In the first sentence with `strong`, `bootcamp` is capital.

File: pydis_site/templates/events/pages/code-jams/8/github-bootcamp.html [hunk 2]
```diff
@@ -13,24 +13,29 @@
...
+        be made available here for download.
     </p>
     <h3>Session 1: Setting up Git</h3>
     <p>
-        This session will run for an hour and be focused on installing and setting up git for the first time.
-        We'll have helpers available to help you set up git and get it integreated with your editor of choice.
+        This first session will run for an hour starting at 4PM UTC. It will be focused on installing and setting up git for the first time.
+        We'll have helpers available to help you set up git and integreated with your editor of choice.
```
Comment (Shivansh-007) [inline pydis_site/templates/events/pages/code-jams/8/github-bootcamp.html]:
```suggestion
        We'll have helpers available to help you set up git and integrated with your editor of choice.
```
Typo

File: pydis_site/templates/events/pages/code-jams/8/github-bootcamp.html [hunk 3]
```diff
@@ -13,24 +13,29 @@
...
         Whether it's PyCharm, VSCode, or vim, we can help you get rolling with version control.
     </p>
     <h3>Session 2: Using git and github in a team environment</h3>
     <p>
-        This session will run for an hour. It will be focused on using git and GitHub in a team environment.
-        You'll be put in a group of people with a repo to work on.
+        This session will run for an hour starting at 5PM UTC. It will be focused on using git and GitHub in a team environment.
+        You'll be put in a group with other attendees and use the common git commands for working in a repo.
+        You'll learn how to set-up a GitHub reposity with other contributors, how to make branches, make commits, pull down changes, and then make pull requests.
```
Comment (Shivansh-007) [inline pydis_site/templates/events/pages/code-jams/8/github-bootcamp.html]:
```suggestion
        You'll learn how to set-up a GitHub repository with other contributors, how to make branches, make commits, pull down changes, and then make pull requests.
```
Typo

**Review by Shivansh-007 — CHANGES_REQUESTED** [review]

**Review by fisher60 — APPROVED** [review]
All looks good. No typos as far as I can tell.

**Pull request merged**